assets/js/fme.js

```diff
@@ -199,21 +199,31 @@ const FME = {
    * Update the visibility of the FME card
    */
   updateVisiblity: function () {
-    $('#fme-container').toggle(Settings.isFmeDisplayEnabled);
+    document.getElementById('fme-container').style.display = Settings.isFmeDisplayEnabled ? '' : 'none';
```

Comment:
I don't think it is correct syntax. Very likely it is recognizeable for most browsers, but i would prefer to use display block or inline.

Reply:
I checked that, and looks it is widely supported